projects/cincinnati/OWNERS

```diff
@@ -0,0 +1,37 @@
+approvers:
+- aditya-konarde
```

Comment:
Pretty sure app-SRE doesn't care about what's happening over in openshift/release.  @aditya-konarde , can you confirm?  Related discussion around why/how app-SRE are approvers for openshift/cincinnati over [here][1].

[1]: https://github.com/openshift/cincinnati/pull/193#discussion_r365363305

Reply:
Having access to this namespace might help with resolving issues on stage. I copied this from cincinnati PR owners, but its not necessary to add app-sre for this to work